src/Validator/Constraints/BonusPointsApply.php

```diff
@@ -9,7 +9,10 @@
 final class BonusPointsApply extends Constraint
 {
     /** @var string */
-    public $message;
+    public $messageInvalidNumber;
```

Comment:
Why both of this fields have no value assigned? I don't see them assigned in the entire class